lib/tools/adb-commands.js

```diff
@@ -532,6 +532,15 @@ methods.processExists = async function (processName) {
   }
 };
 
+/**
+ * Get TCP port forwarding with adb on the device under test.
+ * @return {string} The output of the corresponding adb command. A string contains multiple forwarding lines with line break.
+ */
+methods.getForwardList = async function () {
+  log.debug(`List forwarding ports`);
+  return await this.adbExec(['forward', '--list']);
```

Comment:
Now that I've thought about this a bit, I think this should return an array of strings (so, the output split by newlines). This makes more sense to me.